### DISCUSSION TITLE
下载本地失败，提示S-1-1-0权限错误

### DISCUSSION THREAD

**Discussion by 404gods** [AlistGo/alist#846] [Q&A]
又来问问题了。。。
我用raidrive 挂载了Alist上的天翼云盘做测试，上传基本正常了，下载如果：
1、proxy开、webdav_proxy开、webdav direct关 ，下载报错，需要S-1-1-0权限。
2、proxy关、webdav_proxy关、webdav direct关 ，下载报错，需要S-1-1-0权限。
3、proxy关、webdav_proxy关、webdav direct开 ，下载成功，但是下载速度超级慢，只有几百KB/秒。
夸克云也测试了，不开webdav direct 就下载失败，开了会成功，但是速度同样超级慢。
![photo_2022-03-29_13-31-50](https://user-images.githubusercontent.com/50573302/160540221-af4212eb-33e3-4add-8b60-4ef65e4f81f0.jpg)

**Comment by xhofe** [comment]
302 客户端适配本来就不好没有办法，webdav_direct 直接通过服务器中转返回适配最好。

**Reply by xhofe** [reply]
速度慢取决于你的服务器。

**Reply by 404gods** [reply]
不应该啊，我是安装在本地的群晖，就算中转也不会只有几百KB的速度啊。。。我使用raidrive挂载其他项目比如zxbu的阿里云项目都是满速下载，不过大佬既然说了就这样吧。。。

**Reply by xhofe** [reply]
本地测试开启webdav_direct：
天翼云盘：
![image](https://user-images.githubusercontent.com/36558727/160575265-a861f21a-7737-4d6d-a0ed-7a0c3e47a6e6.png)

阿里云盘：
![image](https://user-images.githubusercontent.com/36558727/160571286-82b988b1-068b-4af6-bdfe-b8829a4416f4.png)
webdav只能单线程

**Reply by 404gods** [reply]
谢谢大佬！重新安装了最新的版本和重新设置，现在夸克网盘及天翼云盘上传下载都正常了。
现在问题只有，天翼云（家庭）上传报错：目标文件夹访问被拒绝，需要权限来执行此操作。
已经开启了webdav direct，正确获取了family id，列表显示正常。请教大佬会是什么原因？

**Reply by xhofe** [reply]
刚刚试了 没发现这个问题。

**Reply by 404gods** [reply]
我试了一下家庭空间的用户名后面带上@189.cn就没报错了，个人空间的反而不需要

**Reply by kekeyo1314** [reply]
> 不应该啊，我是安装在本地的群晖，就算中转也不会只有几百KB的速度啊。。。我使用raidrive挂载其他项目比如zxbu的阿里云项目都是满速下载，不过大佬既然说了就这样吧。。。

进展咋样啊，我也遇到了s-1-1-0的权限问题，直接下载就报错，中转的话我试试

**Comment by 137080487** [comment]
天翼云盘根目录ID是多少啊？我的显示-11 添加不成功啊，提示要二次验证
